aten/src/ATen/native/cuda/BatchLinearAlgebra.cu

```diff
@@ -824,9 +824,27 @@ AT_ERROR("cholesky: MAGMA library not found in "
     }
 
     MAGMAQueue magma_queue(self.get_device());
-    magmaCholeskyBatched<scalar_t>(
-      uplo, n, self_array, n, info_array,
-      batch_size, magma_queue);
+
+    // Compute as many batches of 262140 possible
+    // 262140 is the size of the largest batch of matrices that can be run with
+    // violating maximum kernel configuration
+    // The number of "mini"-batches are floor(batch_size / 262140)
+    // and these cover floor(batch_size / 262140) * 262140 cholesky calls
+    int64_t mini_batches = batch_size / 262140, mini_idx;
```

Comment:
Should/can this magic number be put in some const local variable?

Reply:
I wouldn't call this is a magic number since this is very much dependent on the maximum value of the z-dimension of a grid. And because this is pretty much the only place it's used, I didn't consider initializing a variable as a `constexpr` with this value.

Reply:
it is used all over in this function. above this line, initialize it as `const int64_t z_limit = 262140` and you'll save a refactor bug, or a typo bug from the future